AdminUi/src/AdminUi/ClientApp/src/app/components/quotas/identity/identity-details/identity-details.component.html

```diff
@@ -84,7 +84,9 @@ <h2 class="header-title">{{ header }}</h2>
                 </ng-container>
                 <ng-container matColumnDef="metric">
                     <th mat-header-cell *matHeaderCellDef>Metric</th>
-                    <td mat-cell *matCellDef="let Quota"></td>
+                    <td mat-cell *matCellDef="let Quota">
+                        <span *ngIf="Quota.source === 'Tier'"> Tier quota does not take effect if there is at least one individual quota on the same metric. </span>
+                    </td>
```

Comment:
Let's make one step back and discuss what we actually want. Because I don't think that your implementation matches the expectations.

So the goal is to enable blind people to know why the rows are disabled. For that, we need a text that screen readers are able to parse. Because I don't know if they can read tooltips. 

But you know what? Since we will migrate to Flutter soon, I don't think that we should waste even more time into this. Just undo everything regarding this text.

Reply:
Changes are undone.